YoYoProject/Controllers/GMResource.cs

```diff
@@ -11,5 +12,11 @@ public abstract class GMResource : ControllerBase
         internal Guid ResourceInfoId { get; set; }
 
         internal abstract void Create(string name);
+
+        public void AddResourceToFolder(string filtername)
```

Comment:
I feel like there's a better way to implement this. But I'm too removed from the project to provide any better suggestions. Feels like something that should be handled higher level, at the GMProject level with some abstractions like a flag denoting that these should be added.